css/index.css

```diff
@@ -143,4 +133,63 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+.container2 {
+    width: 800px;
+    margin: 0 auto;
+} 
+.middle-content .boxes {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+
+.middle-content .boxes {
+    width: 12.5%;
+    height: 100px;
+    background: black;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+} 
+.boxes .box1{
+    background:teal;
+}
+.boxes .box2{
+background:gold;
+}
+.boxes .box3{
+    background: cadetblue;
+}
+.boxes .box3{
+    background:coral;
+}
+.boxes .box4{
+    background: crimson;
```

Comment:
Great job on implementing dry programming here 👏 I like how you only targeted individual boxes for the background color and then you applied the same CSS to all of the boxes by targeting them with boxes as your classname. 💯